apps/balancer-tools/src/app/apr/(components)/HeaderEndButton.tsx

```diff
@@ -30,10 +30,9 @@ export default function HeaderEndButton({
     if (!poolId && (!roundId || roundId === "current")) {
       router.push(`/apr/round/${LAST_ROUND_ID}`);
       setSelectedRound(LAST_ROUND_ID);
-    } else {
-      setSelectedRound(undefined);
+    } else if (typeof roundId === "undefined") {
+      setSelectedRound("");
     }
```

Comment:
was this actually causing any issues?

Reply:
It just be like that, Should be fixed by now by [this pr](https://github.com/radix-ui/primitives/issues/1569#issuecomment-1568518196) but that didn't work, so I did the suggested workaround1